modules/tinymce/src/themes/silver/main/ts/ui/dialog/Collection.ts

```diff
@@ -108,7 +129,10 @@ export const renderCollection = (
     dom: {
       tag: 'div',
       // FIX: Read from columns
-      classes: [ 'tox-collection' ].concat(spec.columns !== 1 ? [ 'tox-collection--grid' ] : [ 'tox-collection--list' ])
+      classes: [ 'tox-collection' ].concat(spec.columns !== 1 ? [ 'tox-collection--grid' ] : [ 'tox-collection--list' ]),
+      attributes: {
+        'aria-live': 'polite'
```

Comment:
Just a couple of notes I found about using this `aria-live`:
- All characters that appear in the dialog content, are announced. This may not be the same as the audit's suggestion, which is to announce `how many items there are in the list from the start or if it has filtered out any results upon typing`
- The announcement is cut off early. We might need to escalate the politeness of `aria-live` to `aggressive`.

Reply:
Yeah, this isn't really what we want to do. We're after something a bit more like `modules/alloy/src/main/ts/ephox/alloy/aria/AriaVoice.ts`, where it creates a span and adds it to the page with a specific message, and then deletes it. Because it has the appropriate `aria-live` settings, it is read by a screen-reader. We don't want to rely on it reading the whole collection.